matplotlib/pyplot.pyi

```diff
@@ -126,7 +126,7 @@ def subplots(
 @overload
 def subplots(
     nrows: Literal[1],
-    ncols: Literal[1],
+    ncols: Literal[1] = ...,
```

Comment:
plt.subplots(1, 1)  was working

plt.subplots(1)  now this will also hit this overload

Reply:
I'm not 100% sure this is the proper fix though